hack/validation/requirements.sh

```diff
@@ -2,11 +2,22 @@
 
 # Adding validation for nodeclaim 
 
+# v1beta1
+## checking for restricted labels while filtering out well known labels
+ yq eval '.spec.versions[1].schema.openAPIV3Schema.properties.spec.properties.requirements.items.properties.key.x-kubernetes-validations += [
```

Comment:
Does one of these need to be [0]?
```suggestion
 yq eval '.spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.requirements.items.properties.key.x-kubernetes-validations += [
```

Reply:
No this should be how it is, since that will be index 1 is v1beta1